src/js/references/components/__tests__/ReferenceList.test.tsx

```diff
@@ -57,9 +62,6 @@ describe("<ReferenceList />", () => {
         });
 
         it("should not render creation button when [canCreate=false]", async () => {
-            const permissions = createFakePermissions({ create_ref: true });
-            const account = createFakeAccount({ permissions: permissions });
-            mockAPIGetAccount(account);
```

Comment:
I think we still want to fake the account, just make sure the faked acc doesn't have the request perm. 

**Non-Blocking**: As written this test is a bit iffy since it can't tell the difference between the data not having loaded and the user not having perms. This might be fixable quickly be ensuring the API mocks have been called before checking for the button. Might still end up with a race condition though. Will merge without this being resolved as it is a pre-existing issue

Reply:
I agree; added back the fake acc with no create perms